linux_pipeline/Jenkinsfile_kernel_bisect_controller

```diff
@@ -41,6 +48,14 @@ properties ([
             choices: 'Ubuntu_18.04.1\nCentOS_7.5',
             name: 'DISTRO_VERSION',
             description: 'Distro version to be tested.'],
+        [$class: 'StringParameterDefinition',
+            name: 'GOOD_TRIES',
+            defaultValue: '1',
+            description: 'How many times to try to test a good commit. If set to 2, then it will check a commit, if it is a good commit, it will try a second time to see if it is a good commit and return the second run result.'],
```

Comment:
run case 2 times, if both pass, it is good commit?

Reply:
that is the idea, to hunt for the first bad run of lisav2